pandas/tests/io/formats/test_to_csv.py

```diff
@@ -6,7 +6,7 @@
 import pytest
 from pandas import DataFrame
 from pandas.util import testing as tm
-
+from pandas.util.testing import capture_stdout
```

Comment:
would it be possible to use tm.capture_stdout rather add another import?

Reply:
good point!